homework3/js/main.js

```diff
@@ -0,0 +1,112 @@
+//part1
+//#1
+
+let a = 'none';
+a === 'block' 
+    ? console.log('block')
+    : a === 'none' 
+        ? console.log('none')
+        : a === 'inline' 
+            ? console.log('inline')
+            : console.log('other')
+ 
+let x = '';
+switch(x){
+    case 'block':
+        console.log('block');
+        break
+    case 'none':
+        console.log('none');
+        break
+    case 'inline':
+        console.log('inline');
+        break
+    default: console.log('other');
+}
+//#2
+
+let f = '';
+f === 'hidden'
+    ? [f = 'visible', console.log('visible')]
+    : [f = 'hidden', console.log('hidden')];
+
+let p = '3';
+p === 0
+    ? [p = 1, console.log(p = 1)]
+    : p < 0
+        ? [p = 'less then zero', console.log(p = 'less then zero')]
+        : p > 0
+            ? [p*=10, console.log(p*= 10)]
+            : console.log(p);
+
+//#3
+let car = { name: 'Lexus', age: 6, create: 2008, needRepair: false }
+car.age > 5
+    ? [console.log('Need Repair'), car.needRepair = true]
+    : car.needRepair = false;
+
+//part2
+//#1
+
+let g = 'i am in the easycode'
+let h = '';
+
+for (let i = 0; i < g.length; i++){
+    h += (g [i-1] === ' ')
+    ? g[i].toUpperCase() 
+    : g[i];
+}
+    console.log(h);
+
+//#2
+
+let c = 'tseb eht ma i';
+let d = '';
+ 
+for(let i = c.length-1; i >=0; i--){
+    d += c[i];
+}
+console.log(d);
+
+//#3
+
+function factorial(n){
```

Comment:
it's cool. Hope you did it by yourself and not copied it from internet. Even if you copied then you should understand what this function does. But if by yourself - GOOD FOR YOU)